schema/deploy/functions/pending_new_project_revision_002_create_after_cif_user_update.sql

```diff
@@ -0,0 +1,17 @@
+-- Deploy cif:functions/pending_new_project_revision_002_create_after_cif_user_update to pg
+-- requires: functions/pending_new_project_revision
```

Comment:
It would be good if all the re-create functions had a dependency on their equivalent `...001_drop...`
sqitch executes the plan in order so this ultimately doesn't matter too much, but consistency is nice

```suggestion
-- requires: functions/pending_new_project_revision_001_drop_before_cif_user_update
```